docs/idl/org.wasmcloud/messaging.smithy

```diff
@@ -0,0 +1,119 @@
+// messaging.smithy
```

Comment:
This file appears twice in the pr. Is this because it's getting copied? I don't want it to be hard to find the right one.

Reply:
the docs/ folder is what's published to github pages. It does contain a copy of the true interface file, and github actions should update the docs/idl file whenver it's tagged with a release. I'll add notes to the README to try to clarify.

Reply:
Yeah I figured it was a publication duplicate. Thanks